debezium-connector-ibmi/src/test/java/io/debezium/connector/db2as400/As400ConnectorIT.java

```diff
@@ -37,12 +42,22 @@ public void shouldSnapshotAndStream() throws Exception {
         // Wait for snapshot completion
         var records = consumeRecordsByTopic(1);
 
-        TestHelper.testConnection().execute(
-                "INSERT INTO " + TABLE + " VALUES (2, 'second')",
+        records.print();
+
+        // JdbcConnection conn = TestHelper.testConnection().setAutoCommit(true);
```

Comment:
Please do not modify existng test but intorduce a new one similar to `io.debezium.connector.postgresql.TransactionMetadataIT` to test the transaction handling in detail.